prov/efa/src/efa_domain.c

```diff
@@ -108,6 +118,7 @@ static int efa_domain_init_device_and_pd(struct efa_domain *efa_domain, const ch
 	if (i == g_device_cnt)
 		return -FI_ENODEV;
 
+	EFA_INFO(FI_LOG_DOMAIN, "Domain  %s selected device %s\n", domain_name, device_name);
```

Comment:
Superfluous whitespace?

```suggestion
	EFA_INFO(FI_LOG_DOMAIN, "Domain %s selected device %s\n", domain_name, device_name);
```

Reply:
Thank you! Fixed.